docker-compose.test.yaml

```diff
@@ -14,3 +14,10 @@ services:
       - ./test-output:/home/node/test-output
       - ./jest.config.js:/home/node/jest.config.js
       - ./package.json:/home/node/package.json
+
+  ffc-mpdp-backend-postgres:
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+volumes:
+  postgres_data:
```

Comment:
This is now a necessary step due to recent changes to Docker Compose and how it works with volumes.  Have to declare a volume for the test data.